src/test/java/net/sf/jabref/logic/importer/fetcher/MedlineFetcherTest.java

```diff
@@ -0,0 +1,182 @@
+        entryWijedasa.setField("status", "Publisher");
+        entryWijedasa.setField("title", "Denial of long-term issues with agriculture on tropical peatlands will have devastating consequences.");
+        entryWijedasa.setField("year", "2016");
+
+        entryEndharti = new BibEntry();
+        entryEndharti.setType(BibLatexEntryTypes.ARTICLE);
+        entryEndharti.setField("title", "Dendrophthoe pentandra (L.) Miq extract effectively inhibits inflammation, proliferation and induces p53 expression on colitis-associated colon cancer.");
+        entryEndharti.setField("author", "Endharti, Agustina Tri and Wulandari, Adisti and Listyana, Anik and Norahmawati, Eviana and Permana, Sofy");
+        entryEndharti.setField("created", "2016-9-27");
+        entryEndharti.setField("issn", "1472-6882");
+        entryEndharti.setField("issn-linking", "1472-6882");
+        entryEndharti.setField("issue", "1");
+        entryEndharti.setField("journal", "BMC complementary and alternative medicine");
+        entryEndharti.setField("journal-abbreviation", "BMC Complement Altern Med");
+        entryEndharti.setField("keywords", "CAC; Dendrophtoe pentandra; IL-22; MPO; Proliferation; p53");
+        entryEndharti.setField("nlm-id", "101088661");
+        entryEndharti.setField("owner", "NLM");
+        entryEndharti.setField("pages", "374");
+        entryEndharti.setField("pmid", "27670445");
+        entryEndharti.setField("pubmodel", "Electronic");
+        entryEndharti.setField("pubstatus", "epublish");
+        entryEndharti.setField("revised", "2016-9-27");
+        entryEndharti.setField("status", "Publisher");
+        entryEndharti.setField("volume", "16");
+        entryEndharti.setField("year", "2016");
+
+        bibEntryIchikawa = new BibEntry();
+        bibEntryIchikawa.setType(BibLatexEntryTypes.ARTICLE);
+        bibEntryIchikawa.setField("author", "Ichikawa-Seki, Madoka and Guswanto, Azirwan and Allamanda, Puttik and Mariamah, Euis Siti and Wibowo, Putut Eko and Igarashi, Ikuo and Nishikawa, Yoshifumi");
+        bibEntryIchikawa.setField("chemicals", "Antibodies, Protozoan, Antigens, Protozoan, GRA7 protein, Toxoplasma gondii, Protozoan Proteins");
+        bibEntryIchikawa.setField("citation-subset", "IM");
+        bibEntryIchikawa.setField("completed", "2016-07-26");
+        bibEntryIchikawa.setField("copyright", "Copyright © 2015 Elsevier Ireland Ltd. All rights reserved.");
+        bibEntryIchikawa.setField("country", "Netherlands");
+        bibEntryIchikawa.setField("created", "2015-09-26");
+        bibEntryIchikawa.setField("doi", "10.1016/j.parint.2015.07.004");
+        bibEntryIchikawa.setField("issn", "1873-0329");
+        bibEntryIchikawa.setField("issn-linking", "1383-5769");
+        bibEntryIchikawa.setField("issue", "6");
+        bibEntryIchikawa.setField("journal", "Parasitology international");
+        bibEntryIchikawa.setField("journal-abbreviation", "Parasitol Int");
```

Comment:
Remove abbrevation field, JabRef has own functionality to toggle the abbreviation of journals.